specification/authorization/resource-manager/Microsoft.Authorization/preview/2021-11-16-preview/authorization-AccessReviewCalls.json

```diff
@@ -0,0 +1,2708 @@
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Authorization/accessReviewHistoryDefinitions": {
+      "get": {
+        "tags": [
+          "GetAccessReviewHistoryDefinitions"
+        ],
+        "operationId": "AccessReviewHistoryDefinitions_List",
+        "x-ms-examples": {
+          "GetAccessReviewHistoryDefinitions": {
+            "$ref": "./examples/GetAccessReviewHistoryDefinitions.json"
+          }
+        },
+        "description": "Lists the accessReviewHistoryDefinitions available from this provider, definition instances are only available for 30 days after creation.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/AccessReviewsHistoryFilterParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Describe the result of a successful operation.",
+            "schema": {
+              "$ref": "#/definitions/AccessReviewHistoryDefinitionListResult"
+            }
+          },
+          "default": {
+            "description": "Unexpected error",
+            "schema": {
+              "$ref": "#/definitions/ErrorDefinition"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Authorization/accessReviewHistoryDefinitions/{historyDefinitionId}": {
+      "get": {
+        "tags": [
+          "GetAccessReviewHistoryDefinition"
+        ],
+        "operationId": "AccessReviewHistoryDefinitions_GetById",
+        "x-ms-examples": {
+          "GetAccessReviewHistoryDefinition": {
+            "$ref": "./examples/GetAccessReviewHistoryDefinition.json"
+          }
+        },
+        "description": "Get access review history definition by definition Id",
+        "parameters": [
+          {
+            "name": "historyDefinitionId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The id of the access review history definition."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "AccessReviewHistoryDefinition",
+            "schema": {
+              "$ref": "#/definitions/AccessReviewHistoryDefinition"
+            }
+          },
+          "default": {
+            "description": "Unexpected error",
+            "schema": {
+              "$ref": "#/definitions/ErrorDefinition"
+            }
+          }
+        }
+      },
+      "put": {
```

Comment:
>put [](http://example.com/codeflow?start=7&length=3)

Your PUT API does not have an input body and your example specifies an input body, please add it to match your requirement. You can refer to other PUT APIs in this spec to see how to include it, just make sure to specify the appropriate definition